.editorconfig

```diff
@@ -12,6 +12,11 @@ insert_final_newline = true
 
 [*.{java,kt,kts,scala,rs,xml,kt.spec,kts.spec}]
 indent_size = 4
+# annotation - "./mvnw clean verify" fails with "Internal Error")
+# multiline-if-else - disabled until auto-correct is working properly
+#   (e.g. try formatting "if (true)\n    return { _ ->\n        _\n}")
+# no-it-in-multiline-lambda - disabled until it's clear what to do in case of `import _.it`
+disabled_rules=annotation,multiline-if-else,no-it-in-multiline-lambda
```

Comment:
When a rule is not prefixed with a`namespace:` is it assumed to be in the standard set?

Reply:
Yep, see [this](https://github.com/pinterest/ktlint/pull/503/files#diff-99e7e840295caf6ea84c484c30d73158R244)